src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/Commands/SubmitAzureRmDataLakeAnalyticsJob.cs

```diff
@@ -0,0 +1,201 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.CodeDom;
+using System.IO;
+using System.Management.Automation;
+using Hyak.Common;
+using Microsoft.Azure.Commands.DataLakeAnalytics.Models;
+using Microsoft.Azure.Management.DataLake.AnalyticsJob.Models;
+
+namespace Microsoft.Azure.Commands.DataLakeAnalytics
+{
+    [Cmdlet(VerbsLifecycle.Submit, "AzureRmDataLakeAnalyticsJob"), OutputType(typeof(JobInformation))]
+    public class SubmitAzureDataLakeAnalyticsJob : DataLakeAnalyticsCmdletBase
+    {
+        // internal const string HiveJobWithScriptPath = "Submit job with script path for Hive";
+        internal const string USqlJobWithScriptPath = "Submit job with script path for SQL-IP";
+        internal const string USqlJobParameterSetName = "Submit SQL-IP Job";
+        // internal const string HiveJobParameterSetName = "Submit Hive Job";
+
+        // TODO: Remove this once hive jobs are enabled
+        private SwitchParameter sqlip = true;
+
+        private int _degreeOfParallelism = 1;
+        private int _priority = 1000;
```

Comment:
if these are defined to hold some default value, another way to achieve that is, just have properties with `{ get; set; }` and assign default values for these properties in the class constructor. The code will generally look cleaner